File: prime-number/python/PrimeNumber.py
def main():
    print('Enter a number: ')
    user_input = int(input());

    if (is_prime_number(user_input)):
        print(f"{user_input} is a prime number.")
    else:
        print(f"{user_input} is a not prime number.")

def is_prime_number(number):
    is_prime = True
    for i in range(2, number):
        if (number % i == 0):
            is_prime = False
    return is_prime

if __name__ == '__main__':
    main()

File: prime-number/java/PrimeNumber.java
import java.util.Scanner;

public class PrimeNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a number : ");
        int input = scanner.nextInt();

        if (isPrimeNumber(input))
            System.out.println(input + " is a prime number.");
        else
            System.out.println(input + " is a not prime number.");

        scanner.close();

    }

    static boolean isPrimeNumber(int number) {
        boolean IsPrime = true;
        for (int i = 2; i < number; i++) {
            if (number % i == 0)
                IsPrime = false;
        }
        return IsPrime;
    }

}
